snuba/cli/query_replayer.py

```diff
@@ -170,27 +192,66 @@ def get_version() -> str:
             blob_getter.get_name_diffs(("queries/", f"{results_directory}/"))
         )
 
+    def create_table_from_err(err: ClickhouseError) -> None:
```

Comment:
Nit: I think a better interface for this method would be `create_table(table_name)`. The logic of decoding the error and figuring out the table name should be separate logic.

Reply:
isolated the parsing of the error